pwiz_tools/Skyline/Controls/Startup/ActionBoxControl.cs

```diff
@@ -70,6 +54,8 @@ private void ControlMouseLeave(object sender, EventArgs e)
             if (!string.IsNullOrEmpty(Description))
             {
                 BackColor = Color.Transparent;
+                iconPictureBox.Visible = true;
+                labelDescription.Visible = false;
             }

```

Comment:
It's weird that you're not setting labelDescription.BackColor here, which would be symmetrical with what's in ControlMouseEnter